app/controllers/api/v0/admin/users_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::V0::Admin::UsersController < ApiController
+  before_filter :require_admin
+
+  def index
+    users = User.all.map { |user| UserPresenter.new(user) }
+    render json: AdminReport.new(users: users), serializer: ::Admin::AdminReportSerializer
```

Comment:
For some reason, Rails got really confused and tried looking for Admin within the Api::V0::Admin::UsersController space